### ISSUE TITLE
Exclusion and inclusions lists are changed, they are doing the opposite

### ISSUE THREAD

**Issue by nandorholozsnyak** [rodnansol/spring-configuration-property-documenter#51]
**Describe the bug**
With the newly introduced exclusion and inclusion lists the devs are able to exclude or include specific groups or properties in their final document, but it is broken.

The problem is the following:

```java
private void filterGroupsAndProperties(TemplateCustomization templateCustomization, CombinedInput entry, List<PropertyGroup> groups) {
        propertyGroupFilterService.postProcessPropertyGroups(new PostProcessPropertyGroupsCommand(templateCustomization, groups, entry.getIncludedGroups(), entry.getExcludedGroups(), entry.getIncludedProperties(), entry.getExcludedProperties()));
}

```

It should be:

```java
private void filterGroupsAndProperties(TemplateCustomization templateCustomization, CombinedInput entry, List<PropertyGroup> groups) {
        propertyGroupFilterService.postProcessPropertyGroups(new PostProcessPropertyGroupsCommand(templateCustomization, groups, entry.getExcludedGroups(), entry.getIncludedGroups(), entry.getExcludedProperties(), entry.getIncludedProperties()));
}
```

A few parameters are changed.

**Comment by nandorholozsnyak** [comment]
Fixed in d17a5683763c44659c8af77079c648b6eb83b8fc

**Comment by nandorholozsnyak** [comment]
Fixed with 0.5.0 release.
